src/drivers/console.c

```diff
@@ -46,4 +50,38 @@ void init_console(kernel_heaps kh)
 {
     heap h = heap_general(kh);
     vga_pci_register(kh, closure(h, attach_console));
+    netconsole_register(kh, closure(h, attach_console));
```

Comment:
Here we are allocating the attach_console closure twice, which is not needed. We could use a local variable `console_attach a = closure(h, attach_console)` and then pass `a` to both vga_pci_register() and netconsole_register().